apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -412,7 +412,6 @@ async function updateAssessmentInstanceFromJson(
       {
         assessment_id,
         uid: json.uid,
-        group_name: json.group_name,
```

Comment:
This change is very very not safe; read the above lines and consider what happens when `query === sql. select_assessment_instance_group`.

Reply:
Agree, this one needs to get refactored. We should be running independent queries and not overloading the single call, yeah?

Reply:
Yeah, I think that's the way to go. Break it out into a function if it helps keep control flow clean!

Reply:
Addressed in 01f2075